components/NavigationDrawer/BUILD

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018-present The Material Components for iOS Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//:material_components_ios.bzl",
+     "mdc_public_objc_library",
+     "mdc_objc_library")
+
+licenses(["notice"])  # Apache 2.0
+
+mdc_public_objc_library(
+    name = "NavigationDrawer",
+    sdk_frameworks = [
```

Comment:
Should this library exclude `MDCBottomDrawerHeader.h`? I see it's included separately in the private target below.

Reply:
Done